recipes/oda-api/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "oda-api" %}
+{% set version = "1.1.43" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oda_api-{{ version }}.tar.gz
+  sha256: 87982756ecfe3ed4ba663cc1685b0b788a545223ada85c6e8287f7b12fcd5b7a
+
+build:
+  entry_points:
```

Comment:
Can you add a comment here that this package has scripts and b/c of that cannot be noarch? That is so we can revisit it in the future and, if the scripts are gone, make it noarch.

PS: The scrips are declared here in the package setup.py

https://github.com/oda-hub/oda_api/blob/7890a709ffe6de620011583ed2a18d4bf9b883f5/setup.py#L25

Reply:
We'll try to remove them I think they are obsolete here https://github.com/oda-hub/oda_api/pull/213

Reply:
It is now [removed](https://github.com/oda-hub/oda_api/blob/941a6857c8014882abd9d2a1fa2fab82d4b06f7c/setup.py#L25), only entrypoints remain. Is this sufficient?

Thanks!

Reply:
Cool. In that case you can make it noarch.

Reply:
It is done!